doc/api/packages.md

```diff
@@ -369,6 +369,45 @@ treating the right hand side target pattern as a `**` glob against the list of
 files within the package. Because `node_modules` paths are forbidden in exports
 targets, this expansion is dependent on only the files of the package itself.
 
+### Subpath folder mappings
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/35746
+    description: Runtime deprecation.
+  - version: v14.13.0
+    pr-url: https://github.com/nodejs/node/pull/34718
+    description: Documentation-only deprecation.
+-->
+
+> Stability: 0 - Deprecated: Use subpath patterns instead.
+
+Before subpath patterns were supported, a trailing `"/"` suffix was used to
+support folder mappings:
+
+```json
+{
+  "exports": {
+    "./features/": "./features/"
+  }
+}
+```
+
+_This feature will be removed in a future release._
```

Comment:
What if we include the first example as inline in the text, like:

```suggestion
Before subpath patterns were supported, a trailing `"/"` suffix was used to
support folder mappings, for example like `"./features/": "./features/"`.

This feature is deprecated and will be removed in a future release.
```

@GeoffreyBooth @ljharb let me know if the above can work for you?

Reply:
That’s fine, but what i mean is, it’d be helpful if the full exports algorithm for slashes was available in docs from old versions, even if it’s barely mentioned in the latest docs.

Reply:
I'm not sure I follow, can you provide a suggestion for what you mean?

Reply:
You can view docs on the website for older node versions. Since in, say, node v12.17, slash exports were not deprecated, I’d prefer to be able to view those docs and see fully fleshed out information about slash exports. Most users would view the docs for the latest version, and would see the depreciation notice.

Reply:
The previous documentation did not have a dedicated section for these mappings, it was melded into the main exports documentation. See eg https://nodejs.org/dist/latest-v12.x/docs/api/esm.html#esm_package_entry_points.

The example of this PR as written currently seems to capture your request though, so would you rather not see the change above?

If so, perhaps we can go with Geoffreys suggestion and include the example at the end instead then? Would that work?

Reply:
I mean, I wouldn't consider any of my requests on this PR a blocker even if i had the place to do so :-) in particular, the [module resolution algorithm](https://nodejs.org/dist/latest-v12.x/docs/api/modules.html#modules_all_together) references `PACKAGE_EXPORTS_RESOLVE`, but that doesn't seem to be documented anywhere, and I was hoping to have an example of that that included slash paths.

Reply:
@ljharb `PACKAGE_EXPORTS_RESOLVE` can be found in the ESM resolve spec at https://nodejs.org/dist/latest-v15.x/docs/api/esm.html#esm_resolver_algorithm_specification.

> I mean, I wouldn't consider any of my requests on this PR a blocker even if i had the place to do so

Yes I understand there's no block here I'm just trying to get to the root of the request in a way that works for you both - I still don't understand exactly what documentation you would like to see that this current example is missing.